### PR TITLE
Update Android gradle to 3.0

### PR DESCRIPTION
**Pull request by f2rf2r**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -19,7 +18,7 @@ ext.getLocalProperty = { propertyName ->
 
 android {
     compileSdkVersion 26
-    buildToolsVersion "26.0.2"
+    buildToolsVersion "27.0.0"
```
Comment (shadow7) [inline app/build.gradle]: I think we should be compiling SDK 27 if we're using the new build tools

**Review by shadow7 — APPROVED** [review]

File: app/build.gradle (hunk 1 above)
Comment (f2rf2r) [reply app/build.gradle]: i will revert it to 26. I would rather not target beta versions

File: app/build.gradle [hunk 2]
```diff
@@ -90,54 +89,54 @@ android {
...
-    compile 'com.jakewharton.timber:timber:4.5.1'
-    compile 'com.squareup.picasso:picasso:2.5.2'
+    implementation 'com.jakewharton.timber:timber:4.5.1'
+    implementation 'com.squareup.picasso:picasso:2.5.2'
 
     // Other
-    compile "io.intrepid.commonutils:commonutils:0.2.2"
-    compile 'uk.co.chrisjenx:calligraphy:2.2.0'
+    implementation "io.intrepid.commonutils:commonutils:0.2.2"
+    implementation 'uk.co.chrisjenx:calligraphy:2.2.0'
```
Comment (russhwolf) [inline app/build.gradle:132]: We can probably pull Calligraphy out now

**Review by russhwolf — APPROVED** [review]

File: app/build.gradle (hunk 2 above)
Comment (f2rf2r) [reply app/build.gradle:132]: that will be a separate PR

**Pull request merged**